_positions/EPA-Fellow-Cloud-Architect.md

```diff
@@ -0,0 +1,37 @@
+---
+title: EPA Innovation Fellow - Cloud Architect/Infrastructure Engineer
+permalink:
+team: RaaS
+active: true
+---
+
+*Please Note: This role is for the Environmental Protection Agency's Innovation Fellowship Program, not 18F.*
+
+#Cloud Architect/Infrastructure Engineer
+
+###Role Summary:
+We are looking for a technical leader to seize on a unique opportunity to develop, implement and support both private, public and hybrid cloud hosting services for the organization. You will be providing the technical leadership for the design, planning, integration, testing and delivery of key modernization initiatives in order to promote improvements in the security, agility, responsiveness, and capital efficiency of EPA’s IT infrastructure and customer delivery services.
+
+In addition to having a technical skills and experience in modern cloud computing platforms an ideal candidate must also have a broad understanding of open source software and standards to promote cost-effective adoption of commodity services as well as promote interoperability and automation within and across the full technology stack (applications and infrastructure).
+
+You must also understand enterprise and industry (DevOps) best practices for managing and streamlining change management processes. You will be a critical component of the team responsible for delivering strategic services and deploying capabilities in the cloud. You will work alongside executive leadership as well as our users to provide technical thought leadership to deliver new cloud infrastructure and ensure new and existing applications are matured to include to new cloud infrastructure capabilities.
+
+As an cloud architect and solutions engineer, the you will be responsible for providing the technical support and expertise to analyze requirements, in conjunction with the current network topology and infrastructure architecture to deliver technical solutions to enhance and mature the delivery of agency services, utilizing modern cloud-based services.
+
+###Key Objectives:
+**You will serve as the innovative technical leader in the modernization efforts for the the highly complex and significant systems and services provided by NCC.**
```

Comment:
What's NCC? On first mention of the acronym, please spell it out and append `(NCC)`.